src/utils/axiosUtils.ts

```diff
@@ -0,0 +1,28 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import HttpsProxyAgent from 'https-proxy-agent';
+
+/**
+ * This function is called to better handle axios.get calls with logic that allows
+ * the use of proxies. Not needed in tests unless specifically testing with proxies.
+ * Check https://github.com/axios/axios/issues/3459#issuecomment-766171276 for more info.
+ * @param url {string} - string representation of url to get
+ * @param responseType {any} - optional parameter to change the data type needed from get
+ * (ex. arraybuffer). In default it returns JSON
+ */
+export async function axiosGet(url: string, options?: any): Promise<AxiosResponse<any>> {
+  const httpsProxy = process.env.HTTPS_PROXY;
+  let axiosOptions: AxiosRequestConfig = {};
+  if (options) {
+    axiosOptions = options;
+  }
```

Comment:
I think this can actually be simplified to the following:
```ts
const axiosOptions: AxiosRequestConfig = options ?? {};
```
The `??` operator is a neat little TypeScript trick.  Basically, `A ?? B` means "if A is defined, use A, otherwise use B".

You might have to also change the `options` argument in the signature to `options?: AxiosRequestConfig` -- but if it allows that (without complaining) that's probably better anyway.

Reply:
I think it would be good to do the last thing that @cmoesel mentioned here. I believe you should be able to update the argument to the function to specify the `AxiosRequestConfig` type (so it's `options?: AxiosRequestConfig` like Chris mentioned). This also should mean that you don't need to specify the type when assigning a value to `axiosOptions` because it will get the same type as the option in the function.

I think this is probably preferable so we don't pass in options that aren't supported.

Reply:
Done!